src/DocTests.jl

```diff
@@ -242,8 +242,10 @@ end
 function eval_repl(block, sandbox, meta::Dict, doc::Documenter.Document, page)
     for (input, output) in repl_splitter(block.code)
         result = Result(block, input, output, meta[:CurrentFile])
+        src_lines = Documenter.find_block_in_file(result.block.code, result.file)
         for (ex, str) in Documenter.parseblock(input, doc, page; keywords = false, raise=false)
             # Input containing a semi-colon gets suppressed in the final output.
+            @debug "Evaluating doctest REPL line from $(Documenter.locrepr(result.file, src_lines))" unparsed_string = str parsed_expression = ex
```

Comment:
```suggestion
            @debug "Evaluating REPL line from doctest at $(Documenter.locrepr(result.file, src_lines))" unparsed_string = str parsed_expression = ex
```

Could rephrase a bit to make it clear that the line numbers are for the whole doctest?